src/h5web/BreadcrumbsBar.tsx

```diff
@@ -48,26 +48,17 @@ function BreadcrumbsBar(props: Props): JSX.Element {
           </span>
         </h1>
       )}
-      <div role="tablist" className={styles.modeToggler}>
-        <button
-          type="button"
-          role="tab"
-          className={styles.btn}
-          aria-selected={!isInspecting}
-          onClick={() => onSetInspecting(false)}
-        >
-          Display
-        </button>
-        <button
-          type="button"
-          role="tab"
-          className={styles.btn}
-          aria-selected={isInspecting}
-          onClick={() => onSetInspecting(true)}
-        >
-          Inspect
-        </button>
-      </div>
+      <ToggleGroup
+        role="tablist"
+        ariaLabel="Viewer mode"
+        value={String(isInspecting)}
+        onChange={val => {
+          onChangeInspecting(val === 'true' || false);
```

Comment:
The state is a boolean, but the values can only be string. I was unable to add a generic type to `ToggleGroup` because one cannot create a context with a generic type in React - cf. https://stackoverflow.com/questions/51448291/how-to-create-a-generic-react-component-with-a-typed-context-provider.

I decided it was cleaner to force `value` to be a string rather than to use a union type `string | boolean` and to have to always force-cast the value in `onChange`.